### PR TITLE
add code to be reviewed

### PR DESCRIPTION
**Pull request by divinecharlotte**
I added my to-do list code to be reviewed  as required for the exercise

### CODE REVIEW

File: src/functinalities.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    localStorage.setItem('mylist', JSON.stringify(mylist));
+    getmylist();
+  }
+};
+
+const editmylist = ({ index, event }) => {
+  if (event.target.value === '') return;
+  if (event.key === 'Enter') {
+    mylist[index].description = event.target.value;
+    localStorage.setItem('mylist', JSON.stringify(mylist));
```
Comment (RitobrotoMukherjee) [inline src/functinalities.js:43]: Good job using JS ```destructuring``` and ```map``` in ```addmylist``` and ```deletemylist``` methods respectively.
**Optionally:**, I would advise you put all the **CRUD** related functionalities in a different file and import it from that file. Then your **CRUD** and **UI** files will be different and easy to handle in the future.
The code will be well organized and designed this way.

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- <link rel="stylesheet" href="https://use.fontawesome.com/releases/v6.0.0/css/all.css" integrity="sha384-3B6NwesSXE7YJlcLI9RpRqGf2p/EgVH8BgoKTaUrmKNDkHPStTQ3EyoYjCGXaOTS" crossorigin="anonymous" /> -->
+    <title>todo list</title>
+    <!-- <link rel="stylesheet" href="style.css"> -->
```
Comment (RitobrotoMukherjee) [inline src/index.html]: Please remove the unwanted codes by deleting the commented lines. It'll make the code look better and more readable.

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import _ from 'lodash';// eslint-disable-line
```
Comment (RitobrotoMukherjee) [inline src/index.js]: I cannot see any use of ```loadash``` in your file, you can remove this import. If you are using it somewhere then only you should import that module.

File: webpack.config.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const path = require('path');
+
+module.exports = {
+  mode: 'development',
```
Comment (RitobrotoMukherjee) [inline webpack.config.js]: You can change the mode to ```production``` after deploying the project to ```gh-pages``` from ```main``` branch. Remember project is accessible over the web it is in ```production``` mode

**Review by RitobrotoMukherjee — CHANGES_REQUESTED** [review]
Hi @divinecharlotte 👋 

Great job on your project work. 💯 

## Key Highlights

- No Linters Errors. 👍 
- HTML semantics and CSS best practices followed. 🟢 
- The KISS and DRY principles followed. 🏆 

But there are a few suggestions from my end which will make the project follow more of the ES6 modular structure and YAGNI principle. 💻 

**You can skip the optional changes, for now, based on the time restriction you have, but it's good to have.** 🛸

**Comment by divinecharlotte** [comment]
thank you! am going to improve my code your review is so interesting

**Review by RitobrotoMukherjee — APPROVED** [review]
Hi, @divinecharlotte 👋 

### Status Approved 🥇 

All changes are working fine. There else nothing else to say other than merging. 💯

**Pull request merged**